keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -0,0 +1,757 @@
+
+###### Are there any tests for feature enablement/disablement?
+
+This new `TopologyManager` policy option start from beta stage. The unit test will test whether the configured value of max-allowable-numa-nodes is as expected and whether it is the default recommended value when it is not configured.
+
+<!--
+The e2e framework does not currently support enabling or disabling feature
+gates. However, unit tests in each component dealing with managing data, created
+with and without the feature, are necessary. At the very least, think about
+conversion tests if API types are being modified.
+
+Additionally, for features that are introducing a new API field, unit tests that
+are exercising the `switch` of feature gate itself (what happens if I disable a
+feature gate after having objects written with the new field) are also critical.
+You can take a look at one potential example of such test in:
+https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
+-->
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Rollout or rollout fail do not impact already running workloads, only impact the new workloads.
+
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
+
+###### What specific metrics should inform a rollback?
+
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
+
+We have a metric which records the topology manager admission time: "kubelet_topology_manager_admission_time".
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+Rollout or upgrade do not impact already running workloads. We plan to add an e2e test for this in the furture.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
+No.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
```

Comment:
Since we are going straight to beta, some text is needed here.

Reply:
it should be sufficient to mention the metric (https://github.com/kubernetes/enhancements/pull/4624#discussion_r1627545603) can be used to check if the setting is causing unacceptable performance drops

Reply:
yeah, we have a metric for it, we can mention  it here. updated.